src/Frost.ts

```diff
@@ -9,6 +8,19 @@ export interface Configuration {
   readonly timeout?: number
 }
 
+export interface ClaimAttributes {
```

Comment:
Both of these interfaces are going to change once `frost-api` and `node` are updated to `poet-js` 5.x

If these were the only things being used, what issues were arising?

Reply:
Whenever I am trying to update [Frost-web](https://github.com/poetapp/frost-web/pull/347) to the latest version of `frost-client` it is causing issues with Netlify. @wzalazar and I tried to debug and were able to get Travis passing with some `before_install` commands. But, we didn't have any luck with Netlify.

Reply:
Ok, I think this isn't an issue. Very shortly both of these interfaces are going away.

Reply:
Whenever they do change will they need to be imported into frost-client from poet-js? If so I think we will have the same issue at that point.